### PR TITLE
Auth/feature/get requests

### PR DESCRIPTION
**Pull request by jakobuid**
First merge: https://github.com/WalletConnect/WalletConnectKotlinV2/pull/367

### CODE REVIEW

File: android_core/src/main/kotlin/com/walletconnect/android_core/common/model/type/enums/Tags.kt [hunk 1]
```diff
@@ -30,4 +30,8 @@ enum class Tags(val id: Int) {
 
     SESSION_PING(1114),
     SESSION_PING_RESPONSE(1115),
+
+    AUTH_REQUEST(3000),
```
Comment (TalhaAli00) [inline android_core/src/main/kotlin/com/walletconnect/android_core/common/model/type/enums/Tags.kt:34]: I think tags should be SDK specific. We should probably convert the enum class to a sealed class and the child classes be objects

Comment (Elyniss) [reply android_core/src/main/kotlin/com/walletconnect/android_core/common/model/type/enums/Tags.kt:34]: We have an issue for this https://github.com/WalletConnect/WalletConnectKotlinV2/issues/370

File: authSDK/responder/src/main/kotlin/com/walletconnect/responder/ui/accounts/AccountsFragment.kt [hunk 2]
```diff
@@ -20,6 +20,7 @@ import com.walletconnect.responder.R
 import com.walletconnect.responder.common.ACCOUNTS_ARGUMENT_KEY
 import com.walletconnect.responder.databinding.FragmentAccountsBinding
 import com.walletconnect.responder.ui.events.ResponderEvents
+import com.walletconnect.responder.ui.request.RequestStore
```
Comment (Elyniss) [inline authSDK/responder/src/main/kotlin/com/walletconnect/responder/ui/accounts/AccountsFragment.kt:23]: Unused

File: signSDK/sign/src/main/kotlin/com/walletconnect/sign/storage/sequence/SequenceStorageRepository.kt [hunk 3]
```diff
@@ -273,7 +273,7 @@ internal class SequenceStorageRepository(
     }
 
     @JvmSynthetic
-    fun deleteTempNamespacesByTopicAndRequestId(topic: String, requestId: Long) {
+    fun deleteTempNamespacesByTopicAndRequestId(requestId: Long) {
```
Comment (Elyniss) [inline signSDK/sign/src/main/kotlin/com/walletconnect/sign/storage/sequence/SequenceStorageRepository.kt:276]: I would rename to `deleteTempNamespacesByRequestId`

**Review by TalhaAli00 — APPROVED** [review]

**Review by Elyniss — APPROVED** [review]

**Pull request merged**